40-create-ghcred.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#/bin/sh
```

Comment:
the exclamation mark is needed
```suggestion
#!/bin/sh
```